debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -82,8 +80,7 @@
                                                             .withType(Type.INT)
                                                             .withWidth(Width.SHORT)
                                                             .withImportance(Importance.LOW)
-                                                            .withDescription("The number of attempts in a row that no data are returned from Kafka before recover completes. "
-                                                                    + "The maximum amount of time to wait after receiving no data is (recovery.attempts) x (recovery.poll.interval.ms).")
+                                                            .withDescription("DEPRECATED. Not used anymore.")
                                                             .withDefault(4)
```

Comment:
(see other comment)
Maybe increase this default given the new recovery algorithm.